src/test/java/sign_up/interface_adapters/SignUpViewModelTest.java

```diff
@@ -0,0 +1,149 @@
+package sign_up.interface_adapters;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Can replace with `import static org.junit.Assert.*` so you don't have to do `Assertions.____` every time.

Reply:
Done, thank you!